AdminWebsite/AdminWebsite/ClientApp/src/app/services/audio-link-service.ts

```diff
@@ -57,9 +63,9 @@ export class AudioLinkService {
         throw InvalidParametersError({ cloudRoomName: cloudRoomName, date: date, caseReference: caseReference });
     }
 
-    async getAudioLink(hearingId: string): Promise<string[]> {
+    async getAudioLink(hearingId: string): Promise<HearingAudioRecordingResponse> {
         const response = await this.bhClient.getAudioRecordingLink(hearingId).toPromise();
-        return response.audio_file_links;
+        return response;
```

Comment:
I know it was a promise before but could we change this to an observable? Or is there a specific reason for it to be a promise?